src/data/posts.json

```diff
@@ -1,32 +1,47 @@
 [
   {
-    "title": "New board support: the UP board",
-    "tags": ["announcement"],
-    "text": "The UP board is x86-based with a Raspberry Pi compatible 40-pin connector, designed for 'professional makers' who need a reliable, high-performing device to build and grow their projects.",
-    "link": "https://resin.io/blog/up-board-support/",
-    "image": "https://resin.io/blog/content/images/2017/03/UPSlide3Right-EVT-3.png",
+    "title": "LEGO Macintosh Classic with e‑paper display",
+    "tags": ["project"],
+    "text": "An awesome, detailed community project of a wi-Fi enabled LEGO Macintosh Classic running Docker on a Raspberry Pi Zero with an e‑paper display",
+    "link": "https://jann.is/lego-macintosh-classic/",
+    "image": "https://c2.staticflickr.com/4/3762/33583084842_b2089a98f1_z.jpg",
+    "button": "Check it out"
+  },
+  {
+    "title": "Cellular connectivity with resinOS 2.0, 3G dongles, and SORACOM SIM cards",
```

Comment:
Change title to "Add cellular connectivity to your fleet with resinOS 2.0, 3G dongles, and SORACOM SIM cards"